content/blog/2021-10-05-extending-models-with-more-data.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Adding Data to Build a More Generic Model
+date: 2021-10-05
+description: |
+  You can easily make changes to your dataset using DVC to handle data versioning. This will let you extend your models to handle
```

Comment:
💅 style

Reply:
I updated this to have fewer characters per line.